src/components/AdditionalInformation.jsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import {Form, Field, connect } from "formik";
 import ErrorMsg from "./ErrorMessage";
 import FormContainer from './FormContainer';
-import { formIncomplete } from "./checkFormCompletion";
+import { formIncomplete } from "../utilities/checkFormCompletion";
```

Comment:
if you are really cleaning this up, i would consider the following changes

```js
import { IsFormComplete } from './utilities/FormHelpers';
```

Not Blocking

Reply:
Done